selenium.go

```diff
@@ -271,6 +271,7 @@ type WebDriver interface {
 	// MaximizeWindow maximizes a window. If the name is empty, the current
 	// window will be maximized.
 	MaximizeWindow(name string) error
+	MinimizeWindow(name string) error
```

Comment:
This needs a documentation comment.

https://github.com/golang/go/wiki/CodeReviewComments#doc-comments